src/dfx/src/lib/builders/mod.rs

```diff
@@ -365,6 +365,8 @@ pub struct BuildConfig {
     pub lsp_root: PathBuf,
     /// The root for all build files.
     pub build_root: PathBuf,
+    /// If only a subset of canisters should be built, then canisters_to_build contains these canisters' names.
```

Comment:
So `None` means to build all the canisters?

Reply:
Correct. Expanding the comment to explain that as well.